CONTRIBUTING.md

```diff
@@ -103,20 +97,36 @@ The current code conventions for the source files are as follows:
      * One-line non-documentation comments, intended for schema developers only,
        must use the `// ...` style.
 
+<>
+
+<a name="topic_branches"></a>
+## Topic Branches
+
+If you wish to collaborate on a new feature with other GA4GH members you can ask that a topic branch set be created. This will generally involve the creation of identically named topic branches for each of the schema, compliance and server repos. Since Github does not allow pull requests against branches that do not yet exist, you will have to create an issue asking for the topic branch set to be created.
+
+Once a topic branch set exists, pull requests can be made against it in the usual way. It may also be brought up to date with new changes merged into master by anyone with commit access, if the changes produce merely a fast-forward merge for each constituent branch. However, if changes from the master branch create a new merge commit in or or more of the repositories, that commit needs to be reviewed in a pull request. 
+
+Changes made in a topic branch set can be merged into master by creating and then [resolving in the normal way](#issue_resolution) a pull request against the master branch set, irrespective of ownership by a task-team (see below) or not. 
+
+Topic branch sets that have been merged into master and that are no longer being developed upon should be [deleted](https://github.com/blog/1335-tidying-up-after-pull-requests) (they will still appear in the git history), this can be achieved by consensus of those working on the topic branch set, e.g. a specific task-team. 
+
+### Task Team Topic Branch Sets
+
+Frequently topic branch sets are developed by members working within a task team of the [Data Working Group](http://ga4gh.org). Topic branch sets developed by a task team should be prefixed with the task team's name followed by a dash, e.g. reads-foo, refVar-bar. To enable speedy development, task-teams may set their own rules for contribution to the topic branch sets they own. This allows task-teams to develop features in the view of the larger group, while potentially being unencumbered by the more lengthy process of standard [issue resolution](#issue resolution).
```

Comment:
@benedictpaten could you bold the rule for naming topic branches? This is pretty critical, and is kind of lost in a lot of other, "stating the obvious" documentation.